Gifski/Gifski.swift

```diff
@@ -68,6 +68,10 @@ final class Gifski {
 		progress = Progress(parent: .current())
 
 		let completionHandlerOnce = Once().wrap { [weak self] (_ result: Result<Data, Error>) -> Void in
+			// Ensure libgifski finishes no matter what.
+			try? self?.gifski?.finish()
+			self?.gifski?.release()
```

Comment:
I tried moving the `.release()` call into `.finish()`, but for some reason it doesn't release GifskiWrapper then. Looking at it, it should behave exactly the same, but it doesn't...